### ISSUE TITLE
paper list

### ISSUE THREAD

**Issue by wjgaas** [wjgaas/quantize#1]
Deep learning with limited numerical precision. 2015 IBM
    DoReFa-Net: Training low bit-width convolutional neural networks with low bit-width gradients. 2016
    XNOR-Net: ImageNet Classification using binary convolutional neural networks. 2016 washington
    BC: Binary connect.
    BNN: Binarized Neural Networks. NIPS2016
    Fixed point quantization of deep convolutional networks. 2016
    Hardware-oriented approximation of convolutional neural networks. ICLR2016
    TWNs: Ternary weight networks. NIPS2016 ucas
    Quantized convolutional neural networks for mobile devices. CVPR2016 nlpr
    Flexpoint: an adaptive numerical format for efficient training of deep neural networks. 2017 intel
    INQ: Incremental network quantization, towards lossless CNNs with low-precision weights. ICLR2017 intel labs china
    TTQ: Trained ternary quantization. ICLR2017 stanford
    WRPN: wide reduced-precision networks. 2017 detailed
    A Survey of Model Compression and Acceleration for Deep Neural Networks. 201712
    VNQ: Variational network quantization. ICLR2018
    WAGE: Training and Inference with Integers in Deep Neural Networks. ICLR2018 oral tsinghua
    Clip-Q: Deep network compression learning by In-Parallel Pruning Quantization. CVPR2018 SFU
    LQ-NETs: learned quantization for highly accurate and compact deep neural networks. ECCV2018 Microsoft
    Bi-Real Net: Enhancing the performance of 1-bit CNNs with improved Representational capability and advanced training algorithm. ECCV2018 HKU
    Synergy: Algorithm-hardware co-design for convnet accelerators on embedded FPGAs. 2018 UC Berkeley
    Alternating multi-bit quantization for recurrent neural networks. ICLR2018 alibaba
    Efficient Non-uniform quantizer for quantized neural network targeting Re-configurable hardware. 2018
    ELQ: Explicit loss-error-aware quantization for low-bit deep neural networks. CVPR2018 intel tsinghua
    From Hashing to CNNs: training Binary weights vis hashing. AAAI2018 nlpr
    HAQ: Hardware-Aware automated quantization. NIPS workshop2018 mit
    Heterogeneous Bitwidth Binarization in Convolutional Neural Networks. NIPS2018 microsoft
    HALP: High-Accuracy Low-Precision Training. 2018 stanford
    Mixed Precision Training. ICLR2018 baidu
    PACT: parameterized clipping activation for quantized neural networks. 2018 IBM
    Model Compression via distillation and quantization. ICLR2018 google
    Quantization and training of neural networks for efficient integer-arithmetic-only inference. CVPR2018 Google
    Quantized back-propagation: training binarized neural networks with quantized gradients. ICLR2018
    QUENN: Quantization engine for low-power neural networks. CF18 ACM
    Scalable methods for 8-bits training of neural networks. NIPS2018 intel
    SYQ: learning symmetric quantization for efficient deep neural networks. CVPR2018 xilinx
    TSQ: two-step quantization for low-bit neural networks. CVPR2018
    V-Quant: Value-aware quantization for training and inference of neural networks. ECCV2018 facebook
    UNIQ: Uniform noise injection for non-uniform quantization of neural networks. 2018
    Training a binary weight object detector by knowledge transfer for autonomous driving. 2018
    Training competitive binary neural networks from scratch. 2018
    A white-paper: Quantizing deep convolutional networks for efficient inference. 2018 google
    ACIQ: analytical clipping for integer quantization of neural networks. ICLR2019 Intel
    Per-Tensor Fixed-point quantization of the back-propagation algorithm. ICLR2019
